app/components/RecentEvents.tsx

```diff
@@ -83,6 +83,7 @@ export default class RecentEvents extends TsxComponent<{}> {
           {!!this.recentEvents.length &&
             this.recentEvents.map(event => (
               <EventCell
+                key={event.id.toString()}
```

Comment:
Most events don't have an id. Most do have an _id which is somewhat but not entirely reliable. Event hash is the current state of the art for unique event identification